solidity/dashboard/src/pages/rewards/RewardsOverviewPage.jsx

```diff
@@ -54,19 +98,18 @@ const Balance = ({
       ) : (
         <TokenAmount amount={rewardsBalance} currencySymbol="KEEP" />
       )}
-
-      {/* <div className="flex column wrap ">
-        <SubmitButton onClick={onWithdrawAll}>withdraw all</SubmitButton>
-        <span className="text-validation">
-          The beneficiary account receives all withdrawn rewards.
-        </span>
-      </div> */}
+      {onWithdrawAll && (
+        <SubmitButton
+          onSubmitAction={onWithdrawAll}
+          className="btn btn-primary btn-lg mt-2"
+        >
+          withdraw all
```

Comment:
I'm seeing this button even if there's nothing to withdraw, should we disable it in those cases?

Also, clicking on it changes it to say “Success” even if the process fails (e.g., if I change my address by overriding `yourAddress`, clicking the button won't work; similarly, if I click it and there are no tokens to withdraw, nothing happens). Not sure if that's normal, and not necessarily _hugely_ concerned, but worth a closer look.